cloudbuild.yaml

```diff
@@ -0,0 +1,40 @@
+steps:
+- name: "gcr.io/cloud-builders/go"
+  args: [
+    "get",
+    "-t",
+#    "-u",
+    "-v",
+    "./..."
+  ]
+  env: ["PROJECT_ROOT=github.com/google/trillian"]
+- name: "gcr.io/cloud-builders/go"
+  args: [
+    "get",
+    "./server/trillian_log_server"
+  ]
+  env: ["PROJECT_ROOT=github.com/google/trillian"]
+- name: "gcr.io/cloud-builders/go"
+  args: [
+    "get",
+    "./server/trillian_log_signer"
+  ]
```

Comment:
Why run `go get` for these two binaries? This config doesn't appear to do anything with them.

Reply:
The docker builds in the subsequent steps use these to build container images that are pushed to GCR.

Reply:
Those Dockerfiles run `go get` for these binaries themselves; it doesn't need to be done in advance.
https://github.com/google/trillian/blob/9c0d0c07e9f5dee99b02ae2db7adef4ce03ac9d4/examples/deployment/docker/log_server/Dockerfile#L6

Reply:
Ah, right!

Since you have a CI script, shall I just delete the PR?

Reply:
This might be a cleaner way of doing most of what the script does, since I see you can [use kubectl from this file](https://cloud.google.com/container-builder/docs/configuring-builds/build-test-deploy-artifacts#deploying_artifacts) as well. Would you be up for trying to translate that script into a cloudbuild.yaml file? If not, feel free to delete the PR and I'll take a look at it myself.